examples/pylab_examples/custom_cmap.py

```diff
@@ -55,7 +67,30 @@
 never used.
 
 """
+# Make some illustrative fake data:
+
+x = np.arange(0, np.pi, 0.1)
+y = np.arange(0, 2*np.pi, 0.1)
+X, Y = np.meshgrid(x, y)
+Z = np.cos(X) * np.sin(Y) * 10
+
+
+# --- Colormaps from a list ---
 
+colors = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # R -> G -> B
+n_bins = [2, 5, 10, 100]  # Discretizes the interpolation into bins
+cmap_name = 'my_list'
+f, axs = plt.subplots(1, 4, figsize=(10, 5))
```

Comment:
(thx)

On Tue, Jun 28, 2016 at 10:29 AM, Chris Holdgraf <choldgraf@gmail.com>
wrote:

> hooray that worked!
>
> On Tue, Jun 28, 2016 at 10:27 AM, Eric Firing <notifications@github.com>
> wrote:
>
>> In examples/pylab_examples/custom_cmap.py
>> <https://github.com/matplotlib/matplotlib/pull/6662#discussion_r68802996>
>> :
>>
>> >
>> > +colors = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # R -> G -> B
>> > +n_bins = [2, 5, 10, 100]  # Discretizes the interpolation into bins
>> > +cmap_name = 'my_list'
>> > +f, axs = plt.subplots(1, 4, figsize=(10, 5))
>>
>> The cax kwarg takes an Axes that you have made yourself to hold the
>> colorbar; I think what you want is the ax kwarg, which takes the parent
>> Axes from which space for the colorbar will be taken.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/matplotlib/matplotlib/pull/6662/files/b380fd7bae75b8dc9d12d24977f177175674f317#r68802996>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ABwSHWIznsZA2Ju7IYpNa1D0U5yXuIR4ks5qQVl6gaJpZM4JAULd>
>> .
>>
>
>